apps/next-app/package.json

```diff
@@ -13,22 +13,22 @@
     "@types/node": "20.3.3",
     "@types/react": "18.2.14",
     "@types/react-dom": "18.2.6",
+    "@typescript-eslint/eslint-plugin": "^5.59.0",
```

Comment:
why we need eslint here, a?

Reply:
add eslint để chạy setting eslint. Mình có config eslint ở đây mà

Reply:
we have eslint configurated in the root package.json, a

Reply:
Any response here @KazegamiKuon ?

Reply:
removed